dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/merger/MapMerger.java

```diff
@@ -31,11 +33,7 @@
             return Collections.emptyMap();
         }
         Map<Object, Object> result = new HashMap<Object, Object>();
-        for (Map<?, ?> item : items) {
-            if (item != null) {
-                result.putAll(item);
-            }
-        }
+        Arrays.stream(items).filter(Objects::nonNull).forEach(item -> result.putAll(item));
```

Comment:
`forEach` is designed for side effect, a chained stream seems better to me.

Reply:
> I have a doubt that if the flatMap of a HashMap will change the order of iteration of iterator ?

`HashMap` has no guarantee about its mappings' order, so I think it's not a problem even if `flatMap` will change the order.